doc/source/drivers/vector/pmtiles.rst

```diff
@@ -0,0 +1,337 @@
+.. _vector.pmtiles:
+
+================================================================================
+PMTiles -- ProtoMaps Tiles
+================================================================================
+
+.. versionadded:: 3.8
+
+.. shortname:: PMTiles
+
+This driver supports reading and writing `PMTiles <https://github.com/protomaps/PMTiles>`__
+datasets containing vector tiles, encoded in the MapVector Tiles (MVT) format.
+
+PMTiles is a single-file archive format for tiled data. A PMTiles archive can
+be hosted on a commodity storage platform such as S3, and enables low-cost,
+zero-maintenance map applications that are "serverless" - free of a custom tile
+backend or third party provider.
+
+This driver is compatible with all GDAL
+:ref:`network-based virtual file systems <network_based_file_systems>`
+
+The SRS is always the `Pseudo-Mercator <https://en.wikipedia.org/wiki/Web_Mercator_projection>`__
+(a.k.a Google Mercator) projection, EPSG:3857.
+
+The supported datasets must contain a JSON metadata document
+following the
+`MBTiles specification <https://github.com/mapbox/mbtiles-spec/blob/master/1.3/spec.md#vector-tileset-metadata>`__,
+containing at least the ``vector_layers`` array.
+
+Note that the driver will make no effort of stitching together geometries for
+linear or polygonal features that overlap several tiles. An application that
+wishes to eliminate those interrupts could potentially use the CLIP=NO open
+option to get larger boundaries, and use appropriate clipping graphic primitives
+to hide those discontinuities.
+
+Driver capabilities
+-------------------
+
+.. supports_create::
+
+.. supports_georeferencing::
+
+.. supports_virtualio::
+
+Opening options
+---------------
+
+The following open options are available:
+
+-  .. oo:: CLIP
```

Comment:
Does `CLIP=YES/NO` only apply to writing a pmtiles and not reading? Based on my understanding this should be:

Vector tiles are generally produced with a buffer that provides overlaps between adjacent tiles, and can be used to display them properly, **hence the default behavior of clipping.** When using vector tiles as a vector layer source, like in the OGR vector model, this padding is undesirable.

Reply:
> Does `CLIP=YES/NO` only apply to writing a pmtiles and not reading?

no, CLIP is really a reading / open option. The corresponding creation option is BUFFER documented later in the document.
I'm not totally sure I agree with your change. For something that would display tiles, they would do their rendering of the tile with the extra buffer around it, taking into account line styles etc and then clip to the actual tile extent. So if you use the OGR output for rendering, you don't want OGR to do the clipping itself. But if you want OGR to return features for possibly other purposes than rendering, the clipping is preferable.